src/highdicom/legacy/sop.py

```diff
@@ -454,7 +489,7 @@ def __init__(
             instance_number=instance_number,
             manufacturer=ref_ds.Manufacturer,
             modality=ref_ds.Modality,
-            transfer_syntax_uid=None,  # FIXME: frame encoding
+            transfer_syntax_uid=transfer_syntax_uid,
```

Comment:
Just a though, should the default behaviour be to copy the transfer syntax from the input images, and not re-encode the pixels?

Reply:
I generally agree.

However, this may be a good opportunity to avoid some of the exotic lossless JEPG transfer syntaxes that are not widely supported and thereby facilitate downstream decoding. Thoughts?

Reply:
Are you suggesting that if no transfer syntax is explicitly specified, we selectively re-encode some lossless codecs into a more commonly-supported format?

Reply:
Yes, or just not compress at all by default (which has been the behavior so far)

Reply:
If any of the source images was lossy compressed, we also shouldn't re-compress the enhanced image with a lossy transfer syntax

Reply:
Right, the main thing to avoid would be to decompress lossy data and then recompress losslessly (or store with no compression, as is currently the default) thus giving the impression of lossless compression when in fact information has been lost. That's why I'm suggesting simply leaving the transfer syntax and compression alone by default in those situations unless the user specifically requests otherwise. However I don't see a harm in silently translating between different lossless formats, although it is a bit magical

Reply:
The user can currently not set a lossy transfer syntax. If the pixel data was lossy compressed in the first place, the attribute `LossyImageCompression` should already reflect that (and should stay unchanged even if pixel data is now encoded using a lossless transfer syntax).

The behaviour so far has been to decode the pixel data and re-encode it uncompressed. We can (and probably should) change that behavior upon refactoring of the legacy conversion package as part of https://github.com/MGHComputationalPathology/highdicom/pull/34